.github/workflows/build-and-deploy.yaml

```diff
@@ -0,0 +1,24 @@
+# Generated by cdkactions. Do not modify
```

Comment:
Let's delete these two top lines and sub in:
```
# ========================================
# Note:  If you make changes to this CI/CD, please include someone from DevOps in the list of reviewers for the PR.
# ========================================
```

Reply:
Addressed